### PR TITLE
documented StyleBoxFlat

### PR DESCRIPTION
**Pull request by toger5**
okay this time it is a much simpler pr...

### CODE REVIEW

File: doc/base/classes.xml [hunk 1]
```diff
@@ -49206,10 +49206,32 @@
 </class>
 <class name="StyleBoxFlat" inherits="StyleBox" category="Core">
 	<brief_description>
-		Stylebox of a single color.
+		Customizable Stylebox with a given set of parameters. (no texture required)
 	</brief_description>
 	<description>
-		Stylebox of a single color. Displays the stylebox of a single color, alternatively a border with light/dark colors can be assigned.
+		This stylebox can be used to achieve all kinds of looks without the need of a texture.
```
Comment (akien-mga) [inline doc/base/classes.xml]: Keep in mind that every line in the `<description>` will be converted to its own paragraph in HTML (`<p>`), apart from the `[codeblock]`s which are processed as one.
So please avoid empty lines or unnecessary line breaks.

File: doc/base/classes.xml [hunk 2]
```diff
@@ -49206,10 +49206,32 @@
...
+		Those properties are coustomizable:
```
Comment (akien-mga) [inline doc/base/classes.xml]: Typo, c**u**stomizable.

File: doc/base/classes.xml [hunk 3]
```diff
@@ -49453,46 +49475,77 @@
...
 		</member>
 		<member name="border_width_top" type="int" setter="set_border_width" getter="get_border_width" brief="">
+			Border width for the top border.
 		</member>
 		<member name="corner_detail" type="int" setter="set_corner_detail" getter="get_corner_detail" brief="">
+			This sets the amount of vertices used for each corner. Higher values result in rounder corners but take more processing power to compute.
+			By choosing in value you should take the corner radius ([method set_corner_radius]) into account.
+			For corner radius smaller than 10: 4-5 should be enough
+			For corner radius smaller than 30: 8-12 should be enough
+
```
Comment (akien-mga) [inline doc/base/classes.xml]: Same issue here, this line and the new ones could be removed, they'll just make two empty paragraphs for no gain.

Comment (toger5) [reply doc/base/classes.xml]: changed to use much less lines

File: doc/base/classes.xml (hunk 2 above)
Comment (toger5) [reply doc/base/classes.xml]: fixed

File: doc/base/classes.xml (hunk 1 above)
Comment (toger5) [reply doc/base/classes.xml]: also put in one line

**Pull request merged**